.pre-commit-config.yaml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Looks good to me. Here is an example pre-commit-config we've used in the past, if you want to add some additional checks later on. https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/.pre-commit-config.yaml

but for now I think this looks good.